gulpfile.js

```diff
@@ -170,6 +172,7 @@ gulp.task("amd_bundles_src", function (done) {
             'colorbrewer': '../bower_components/colorbrewer/colorbrewer',
             'd3.layout.cloud': '../bower_components/d3-cloud/d3.layout.cloud',
             'font-awesome': '../bower_components/font-awesome/css/font-awesome',
+            'amcharts': '../bower_components/amcharts/dist/amcharts',
```

Comment:
FYI - The paths here should be the same as the config.js (but don't change it now, as I am removing this list from the gulp file.